cms/core/blocks/panels.py

```diff
@@ -3,27 +3,44 @@
 from wagtail import blocks
 
 
-class PanelBlock(blocks.StructBlock):
-    """DS Panel block.
-    https://service-manual.ons.gov.uk/design-system/components/panel
-    https://service-manual.ons.gov.uk/design-system/components/announcement-panel
-    https://service-manual.ons.gov.uk/design-system/components/success-panel
-    https://service-manual.ons.gov.uk/design-system/components/warning-panels.
-    """
-
-    variant = blocks.ChoiceBlock(
-        choices=[
-            ("warn", _("Warning")),
-            ("info", _("Information")),
-            ("announcement", "Announcement"),
-            ("error", "Error"),
-            ("success", "Success"),
-        ],
-        default="warn",
-    )
+class WarningPanelBlock(blocks.StructBlock):
     body = blocks.RichTextBlock(features=settings.RICH_TEXT_BASIC)
-    title = blocks.CharBlock(required=False, label=_("Title (optional)"))
+
+    def get_context(self, value: dict, parent_context: dict | None = None) -> dict:
+        context: dict = super().get_context(value, parent_context)
+        context["variant"] = "warn"
+        return context
+
+    class Meta:
+        template = "templates/components/streamfield/warn_announcement_panel.html"
+        label = _("Warning Panel")
+        group = _("Panels")
+
+
+class InformationPanelBlock(blocks.StructBlock):
+    title = blocks.CharBlock(required=True, label=_("Title"))
+    body = blocks.RichTextBlock(features=settings.RICH_TEXT_BASIC)
+
+    def get_context(self, value: dict, parent_context: dict | None = None) -> dict:
+        context: dict = super().get_context(value, parent_context)
+        context["variant"] = "info"
```

Comment:
Nitpick: as this has it's own template, we don't need to pass the variant through here - it will always be 'info'

Reply:
Very true. I guess my brain was on some weird autopilot/ copy-paste mentality from one class to another haha. Or I think I was doing something then I forgot to remove it, I can't remember. Removed!